packages/teams-js/src/private/externalAppCardActionsForCEA.ts

```diff
@@ -84,7 +84,9 @@ export namespace externalAppCardActionsForCEA {
         ApiName.ExternalAppCardActionsForCEA_ProcessActionSubmit,
       ),
       ApiName.ExternalAppCardActionsForCEA_ProcessActionSubmit,
-      [appId, conversationId, actionSubmitPayload],
+      appId,
```

Comment:
`sendAndUnwarp` will wrap those items in one array. With sending ` [appId, conversationId, actionSubmitPayload]` will result to `[ [appId, conversationId, actionSubmitPayload]]` in hub-sdk